src/jmh/java/net/sf/jabref/benchmarks/Benchmarks.java

```diff
@@ -26,8 +30,10 @@
 @State(Scope.Thread)
 public class Benchmarks {
 
-    String bibtexString;
-    BibDatabase database = new BibDatabase();
+    private String bibtexString;
+    private final BibDatabase database = new BibDatabase();
+    private final List<String> latexConversionStrings = new ArrayList<>();
```

Comment:
You are right... Forgot to do that part... Anyway, maybe better to have a single longer hard-coded string.